### PR TITLE
Dependency and version bump

### PR DESCRIPTION
**Pull request by exsilium**
Hi! Keeping up with the deps 😄  Please pull and publish 🙇  🍻

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,11 +13,11 @@
     "email" : "me@daniel.gs"
   },
   "dependencies": {
-    "lodash": "3.0.x"
+    "lodash": "^4.17.4"
```
Comment (magopian) [inline package.json]: I believe it should now be updated to at least `4.17.5`, according to https://nodesecurity.io/advisories/577

**Review by magopian — COMMENTED** [review]
running `npm audit` gives a security warning about lodash.

File: package.json (hunk 1 above)
Comment (cadesalaberry) [reply package.json]: Correct

Comment (exsilium) [reply package.json]: Caret inclusion takes care of pulling patch and minor releases. Nevertheless, I'll update this pull request in the hopes of eventual merge :crossed_fingers: :wink:

**Comment by skullmasher** [comment]
Hello, the lodash dependency should be up furthermore to >=4.17.13. After [the new CVE](https://snyk.io/vuln/SNYK-JS-LODASH-73638) who impact project such as [nightmareJS](https://github.com/segmentio/nightmare).

Thank you for your work.

**Comment by exsilium** [comment]
The dependency updates and most importantly liberal inclusion of lodash via caret change was done outside the scope of this PR and fixed with #3 . I'll just go ahead and close this PR. Cheers! :tada:

**Pull request closed without merging**